src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LoggerWrapper.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+public class LoggerWrapper : ILspLogger, ILogger
```

Comment:
Hooray for unified logging 🎉 I think a comment above the class on why we have two separate loggers might be useful. I at least was originally confused on first glance 😅